waspc/headless-test/start.js

```diff
@@ -0,0 +1,32 @@
+const cp = require('child_process');
```

Comment:
This file is still pretty misterious to me, I think it needs some cleaning up / better comments @craigmc08 .